.golangci.yml

```diff
@@ -81,6 +81,7 @@ issues:
   exclude:
     # Allow package logger variables (for now)
     - \`logger\` is a global variable
+    - \`testLogger\` is a global variable
```

Comment:
you can just add a `// nolint` directive next to the offenders

Reply:
@llorllale Done